src/modules/modal/modalpopup.js

```diff
@@ -1,5 +1,8 @@
-// import getComments from './getComments';
+/* eslint-disable */
```

Comment:
I would recommend you to don't disable linters, those are important to write cleaner code.